### ISSUE TITLE
R40 scores on KITTI3D / weight sharing

### ISSUE THREAD

**Issue by aboulch** [chaytonmin/Voxel-MAE#16]
Hello,
Thanks for releasing the code of Voxel-MAE.
In the paper I see only R11 score for 3D detection on KITTI3D, would it be possible to share also the R40 scores (as it is now the official metric for KITTI3D) for SECOND and PV-RCNN?
If not possible, could you share model weights (either pre-trained or downstream), so that I can get these R40 scores?
Best regards,

**Comment by chaytonmin** [comment]
> Hello, Thanks for releasing the code of Voxel-MAE. In the paper I see only R11 score for 3D detection on KITTI3D, would it be possible to share also the R40 scores (as it is now the official metric for KITTI3D) for SECOND and PV-RCNN? If not possible, could you share model weights (either pre-trained or downstream), so that I can get these R40 scores? Best regards,

We will provide the results later.
